packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -236,19 +175,6 @@ registerStyles(
       font-weight: 400;
     }
 
-    /* Slotted icons */
-    [part='input-field'] ::slotted(vaadin-icon) {
-      color: var(--lumo-contrast-60pct);
-      width: var(--lumo-icon-size-m);
-      height: var(--lumo-icon-size-m);
-    }
-
-    /* Vaadin icons are based on a 16x16 grid (unlike Lumo and Material icons with 24x24), so they look too big by default */
-    [part='input-field'] ::slotted(vaadin-icon[icon^='vaadin:']) {
-      padding: 0.25em;
-      box-sizing: border-box !important;
-    }
```

Comment:
This CSS is moved here:
https://github.com/vaadin/web-components/blob/91bfb8acc231ec7747bbebbfe2add5b83e647e00/packages/input-container/theme/lumo/vaadin-input-container-styles.js#L81-L91